{{cookiecutter.repository_name}}/operations/train.py

```diff
@@ -73,35 +75,35 @@ def train(config):
             optimizer.zero_grad()
 
             train_metrics["loss"].update_(loss)
-            train_metrics["accuracy"].update_(examples, predictions)
+            train_metrics["standard"].update_(predictions, examples)
```

Comment:
What do we think about the name `standard`? The downside is it's not very descriptive, but I struggle to come up with something that is. `from_predictions_and_examples` is a suggestion but it's a little long